tests/components/airvisual/test_config_flow.py

```diff
@@ -42,6 +56,29 @@
         assert result["errors"] == {CONF_API_KEY: "invalid_api_key"}
 
 
+async def test_migration_1_2(hass):
+    """Test migrating from version 1 to version 2."""
+    conf = {
+        CONF_API_KEY: "abcde12345",
+        CONF_GEOGRAPHIES: [{CONF_LATITUDE: 51.528308, CONF_LONGITUDE: -0.3817765}],
+    }
+
+    config_entry = MockConfigEntry(
+        domain=DOMAIN, version=1, unique_id="abcde12345", data=conf
+    )
+    config_entry.add_to_hass(hass)
+
+    await async_migrate_entry(hass, config_entry)
```

Comment:
Don't call this directly, instead up airvisual and ensure there are 2 config entries afterwards.